tests/functions/test_pivot_longer.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 from pandas.testing import assert_frame_equal
 
-from janitor import patterns
+from re import compile as patterns
```

Comment:
My opinion is that for internal uses (of `patterns`), e.g. in tests, we don't have to follow any deprecation cycle or the like.
I would suggest to remove all uses of `patterns` and change them to `re.compile` directly.

The code will be clearer for developers to read in the future as well. (They might wonder why we choose to perform this strange alias)

Reply:
Agreed, thanks for raising, @thatlittleboy!